src/styles/_catalog.scss

```diff
@@ -1,9 +1,20 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
More correctly to move repeated values to variables. To change the width of the card, it will be enough to change the value of the variable, and not search through the entire code